janMarkusMilan/assignments/slidingwindow/lc209/lc209.js

```diff
@@ -1,5 +1,54 @@
+// Problem Statement #
+
+// Given an array of positive numbers and a positive number ‘S,’ find the length of the smallest contiguous subarray whose sum is greater than or equal to ‘S’. Return 0 if no such subarray exists.
+
+/*
+Edge cases: return 0
```

Comment:
Love that you considered edge cases! It would be better to list out what the edge cases are, and how to handle each one. That way it will be clearer to the interviewer.